Source/Core/Core/HW/WiimoteEmu/Attachment/Nunchuk.cpp

```diff
@@ -93,18 +94,9 @@ void Nunchuk::GetState(u8* const data)
 	s16 accel_y = (s16)(4 * (accel.y * ACCEL_RANGE + ACCEL_ZERO_G));
 	s16 accel_z = (s16)(4 * (accel.z * ACCEL_RANGE + ACCEL_ZERO_G));
```

Comment:
Am I missing something? The `4 *` seems pointless given that there's a `>> 2` below.